### DISCUSSION TITLE
General FAQ/Support and CoC/Guidelines

### DISCUSSION THREAD

**Discussion by televandalist** [RetroAchievements/RAWeb#1469] [Ideas]
I'd like to start moving a few sections from the docs to their own pages on site. 

Pages would include:
- RetroAchievements Manifesto
- User Code of Conduct
- Developer Code of Conduct
- Global Leaderboard and Achievement Hunting Rules
- FAQ

These pages would be basic enough that I could probably put them together, or at least get them started. I would just need someone to test locally since I don't have an environment set up.

**Comment by wescopeland** [comment]
I am happy to help.

**Reply by luchaos** [reply]
Heads up: those pages can be rendered as markdown just like other views. Might make it easier to port and maintain instead of rewriting it to HTML.

**Reply by televandalist** [reply]
Well that will make this a lot easier
